unturned/OpenMod.Unturned/Players/Movement/Events/PlayerMovementEventsListener.cs

```diff
@@ -31,13 +31,13 @@ public override void Unsubscribe()
         public override void SubscribePlayer(Player player)
         {
             player.stance.onStanceUpdated += () => OnStanceUpdated(player);
-            player.animator.onGestureUpdated += () => OnGestureUpdated(player);
+            player.animator.onGestureUpdated += (gesture) => OnGestureUpdated(player, gesture);
```

Comment:
`onGestureUpdated` event does not include a punch gesture. See: https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/1927

Reply:
And also many gestures not included: Facepalm, Point, Wave, and etc..

Reply:
Fixed by #203